numpy/__init__.pyi

```diff
@@ -689,7 +691,8 @@ __all__ = [  # noqa: RUF022
     "array2string", "array_str", "array_repr", "set_printoptions", "get_printoptions",
     "printoptions", "format_float_positional", "format_float_scientific", "require",
     "seterr", "geterr", "setbufsize", "getbufsize", "seterrcall", "geterrcall",
-    "errstate",
+    "errstate", "_get_preserve_0d_arrays", "_set_preserve_0d_arrays"
```

Comment:
```suggestion
    "errstate", "_get_preserve_0d_arrays", "_set_preserve_0d_arrays",
```

Reply:
Let's try, I am worried some auto-formatter will come around and insist that every entry must be on it's own line then, but probably we don't have that yet, heh.